src/main/java/frc/robot/util/MotionMagicPreferenceConstants.java

```diff
@@ -0,0 +1,95 @@
+package frc.robot.util;
+
+import java.util.function.Consumer;
+
+import frc.robot.util.preferenceconstants.DoublePreferenceConstant;
+
+/**
+ * Class used for MotionMagic control, using Velocity, Acceleration, and Jerk
+ */
+
+public class MotionMagicPreferenceConstants {
```

Comment:
This is a really good idea. Can it be expanded to include a PIDPreferenceConstants object? We always want one of those when configuring motion magic.